pyproject.toml

```diff
@@ -32,7 +32,8 @@ dependencies = [
     "debugpy>=1.2.1",
     "requests>=2.28.1",
     "typer>=0.9, <1.0",
-    "rich>=13.0, <14.0"
+    "rich>=13.0, <14.0",
+    "omegaconf>=2.3.0",
```

Comment:
do we want to add an upper version to be safe?

Reply:
Would also just be 2.3.0 here, right? But of course, I can add it 😅